core/shared/src/main/scala/zio/prelude/Equal.scala

```diff
@@ -266,6 +274,13 @@ object Equal extends Lawful[Equal] {
   implicit def EitherEqual[A: Equal, B: Equal]: Equal[Either[A, B]] =
     Equal[A] either Equal[B]
 
+  /**
+   * `Equal` instance for `Float` for imprecise equality with configurable tolerance.
+   */
+  def FloatEqualWithEpsilon(epsilon: Float = 1.0f / 1024 * 1024): Equal[Float] = { (l: Float, r: Float) =>
```

Comment:
I think there is an order of operations issue with the default values here and above. Don't you want this to be `1.0f / (1024 * 1024)`? Otherwise the default value is just `1.0f` which seems like way to high a tolerance.

Reply:
of course! thank you :+1: